assets/scss/base/_lists.scss

```diff
@@ -14,6 +25,26 @@ ul, nav ul {
 	}
 }
 
+/*
+Numbered list
+
+An ordered list with numbers where the parent element uses the modifier `.list--numbered` and the child elements use the class `.numbered`
+
+Markup:
```

Comment:
What a mess!! :cry: 

So I think it would be better to make the example markup a regular `ul` and reach out to those teams, asking them to justify or update their markup.

My concern with displaying unsemantic markup for the examples is that it'll be copy/pasted. If we start with the intended markup in our examples we can contain the nonsense and ask offending frontends to update.